services/web-app/components/type/type-scale.module.scss

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/type' as type;
+
+$type-scale: 0.75rem, 0.875rem, 1rem, 1.125rem, 1.25rem, 1.5rem, 1.75rem, 2rem, 2.25rem, 2.625rem,
+  3rem, 3.375rem, 3.75rem, 4.25rem, 4.75rem, 5.25rem, 5.75rem;
+
+@include breakpoint.breakpoint-down('lg') {
+  .type-scale {
+    overflow-x: scroll;
+  }
+}
+
+.type-scale {
+  margin-top: spacing.$spacing-07;
+  overflow-x: auto;
+
+  table {
+    background: theme.$layer-01;
+  }
+
+  table tbody tr {
+    border: none;
+  }
+
+  table thead th {
+    @include type.type-style('label-01');
+
+    padding: spacing.$spacing-05 0 spacing.$spacing-05 spacing.$spacing-05;
+    border: none;
+    color: theme.$text-secondary;
+    text-align: left;
+  }
+
+  table tbody td {
+    padding-right: spacing.$spacing-05;
+    padding-bottom: auto;
+    padding-left: spacing.$spacing-05;
+    border: none;
+    color: theme.$text-primary;
+    vertical-align: top;
+    white-space: nowrap;
+  }
+
+  table tbody td:first-child {
+    padding: 0 spacing.$spacing-09 0 spacing.$spacing-05;
+    font-size: 0.75rem;
+
+    @include breakpoint.breakpoint('md') {
+      padding-right: 112px;
+    }
+
+    @include breakpoint.breakpoint('lg') {
+      padding-right: spacing.$spacing-07;
+    }
+
+    @include breakpoint.breakpoint('xlg') {
+      padding-right: spacing.$spacing-09;
+    }
+  }
+
+  @for $i from 1 through length($type-scale) {
+    table tbody tr:nth-child(#{$i}) td:nth-child(2) {
+      width: 100%;
+      /* stylelint-disable scss/no-global-function-names */
+      font-size: nth($type-scale, $i);
```

Comment:
```scss
@use 'sass:list' as list;

font-size: list.nth($type-scale, $i);
```

Reply:
This isn't working at all now with the latest update 😿

Reply:
I'm going to simplify this file, having it set up this way doesn't make it any easier to understand really.

Reply:
nvm, ignore me 😆